moto/iot/models.py

```diff
@@ -495,6 +551,12 @@ def describe_thing_type(self, thing_type_name):
             raise ResourceNotFoundException()
         return thing_types[0]
 
+    def describe_endpoint(self, endpoint_type):
+        self.endpoint = FakeEndpoint(endpoint_type, self.region_name)
```

Comment:
Can we assign this on initialization, and just return it here?
The `if not` seems definitely unnecessary

Reply:
Thinking better to follow other modules with a Object ( class ) to handle the returned values.
Remove if not.

Reply:
> Can we assign this on initialization, and just return it here?
> The `if not` seems definitely unnecessary

Move the assignment here, with using _random_string without put it into uilts.py which may cause other functions failures.
Remove if not.

Reply:
I think the class object needs to be there for endpoint since it will call return obj.to_dict() as others, will keep consistency in case there may be some usages incompatibilities ?
remove `if not`